packages/terra-clinical-result/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Unreleased
 
 ### Changed
 * Update docs to use mdx syntax.
+* ClinicalResultBloodPresure no longer truncates in FlowsheetResultCell
```

Comment:
Since the related issue is marked as a bug this changelog entry should probably move under the Fixed section

Reply:
Fixed here https://github.com/cerner/terra-clinical/pull/637/commits/788a6eed719217b512d85dbc8c1fa1c37d747e50#diff-534a7c4c0b2429787b66bb7aac9cade0R10